static/js/directives/ui/repo-tag-history.js

```diff
@@ -31,12 +35,17 @@ angular.module('quay').directive('repoTagHistory', function () {
         };
 
         ApiService.listRepoTags(null, params).then(function(resp) {
+          $scope.cachedFullTags = resp.tags;
           processTags(resp.tags);
         });
       };
 
       $scope.$watch('isEnabled', loadTimeline);
       $scope.$watch('repositoryTags', loadTimeline);
+      $scope.$watch('options.showFuture', function() {
```

Comment:
Prefer not to use `$scope.$watch()`, but this is fine for now.

Reply:
Angular 1!